src/map_loader.cpp

```diff
@@ -63,36 +62,13 @@ namespace
 // Case-unsensitive strings equality-comparision
 static bool StringEquals( const char* const s0, const char* const s1 )
 {
-	unsigned int i= 0;
-	while( s0[i] != '\0' && s1[i] != '\0' )
-	{
-		if( std::tolower( s0[i] ) != std::tolower( s1[i] ) )
-			return false;
-		i++;
-	}
-
-	return std::tolower( s0[i] ) == std::tolower( s1[i] );
+	return ( ::strcasecmp( s0, s1 ) == 0 );
```

Comment:
I am afraid `strcasecmp` can't be used, because it is non-standard.

Reply:
The function is available with  #define _GNU_SOURCE
I've added _MSC_VER windows defines too.